monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1359 @@
+)
+
+replication_lag = mongodb_timeseries(
+    "Replication Lag",
+    [
+        Target(
+            expr=MongoDBMetrics.replication_lag(),
+            legendFormat="{{pod}} <- {{name}}",
+        )
+    ],
+    unit=UNITS.SECONDS,
+    thresholds=THRESHOLDS["replication_lag"],
+    lineInterpolation="smooth",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="always",
+    decimals=2,
+    valueMin=0,
+    description="MongoDB replication lag occurs when the secondary node cannot replicate data fast enough to keep up with the rate that data is being written to the primary node.",
+)
+
+flow_control = mongodb_timeseries(
+    "Flow Control",
+    [
+        Target(
+            expr=f'max(rate(mongodb_ss_flowControl_isLaggedCount{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval]))',
+            legendFormat="fc_count",
+        ),
+        Target(
+            expr=f'max(rate(mongodb_ss_flowControl_isLaggedTimeMicros{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval]))',
+            legendFormat="fc_time",
+        ),
+    ],
+    unit="µs",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    decimals=2,
+    valueMin=0,
+    description="Flow Control is a mechanism that helps to prevent the primary node from being overwhelmed by the number of operations it needs to process. It is used to control the rate at which the primary node can write to the secondary node.",
+)
+
+# Performance Metrics Panels
+query_exec_times = mongodb_timeseries(
+    "Query execution times",
+    [
+        Target(
+            expr=f'avg by (op_type) (rate(mongodb_ss_opLatencies_latency{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval]) / (rate(mongodb_ss_opLatencies_ops{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval]) > 0))',
+            legendFormat="{{op_type}}",
+        )
+    ],
+    unit="µs",
+    thresholds=THRESHOLDS["query_exec"],
+    lineInterpolation="smooth",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    decimals=2,
+    valueMin=0,
+    description="Average latency of operations (classified by read, write, or (other) command)",
+)
+
+queued_ops = mongodb_timeseries(
+    "Queued Operations",
+    [
+        Target(
+            expr=f'max by (type) (mongodb_mongod_global_lock_current_queue{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}})',
+            legendFormat="{{type}}",
+        )
+    ],
+    unit=UNITS.OPS_PER_SEC,
+    thresholds=THRESHOLDS["queue_ops"],
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    decimals=2,
+    valueMin=0,
+    description="Operations queued due to a lock.",
+)
+
+# Additional Performance Panels
+op_queue_size = mongodb_timeseries(
+    "Operation queue size due to locks",
+    [
+        Target(
+            expr=f'sum by(job) (mongodb_ss_globalLock_currentQueue{{count_type=~"readers|writers", job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}})',
```

Comment:
looks pretty similar to the previous panel's metric (`mongodb_ss_globalLock_currentQueue` vs `mongodb_mongod_global_lock_current_queue`, except for the "by type") : what is the difference? how do they relate?
If there is some aggregation (implicit, on mongo side), may be better to make this clear in the title: maybe something like "Queued operations by type" and "Total queued operations"